src/path_manager.c

```diff
@@ -918,81 +936,55 @@ static void family_appeared(struct l_genl_family_info const *info,
                   kernel after mptcpd has started.
```

Comment:
Thanks for pointing that out.  I'll update the inconsistent comments, and skim through the code again in case there are other instances.

I switched over to "MPTCP" instead of "`mptcp`" since the latter is the genl family name used by the multipath-tcp.org kernel, i.e. `MPTCP_GENL_NAME`.  Upstream's genl family name, `MPTCP_PM_NAME`, is "`mptcp_pm`".  "MPTCP" was intended to be more generic.